message/models.py

```diff
@@ -0,0 +1,56 @@
+from django.db import models
+from django.utils import timezone
+from dogowner.models import DogOwner
+from daycare.models import DayCare
+
+
+class AuthorOptions(models.TextChoices):
+    DogOwner = 'O'
+    DayCare = 'D'
+
+
+class Message(models.Model):
+    author = models.CharField(max_length=1,
+                              choices=AuthorOptions.choices)
+    dogowner_id = models.ForeignKey(DogOwner, on_delete=models.CASCADE)
+    daycare_id = models.ForeignKey(DayCare, on_delete=models.CASCADE)
+    text = models.TextField()
+    date = models.DateTimeField(default=timezone.now)
+
+    def __str__(self):
+
+        if(self.author == AuthorOptions.DayCare):
+            res = f"From: {self.daycare_id}"
+            res += f" To: {self.dogowner_id}"
+        else:
+            res = f"From: {self.dogowner_id}"
+            res += f" To: {self.daycare_id}"
+
+        res += f" text: {self.text}"
+        res += f" date: {self.date}"
+        return res
+
+    def create(author, dogowner_id, daycare_id, text):
+
+        new_message = Message(author=author,
+                              dogowner_id=dogowner_id,
+                              daycare_id=daycare_id,
+                              text=text)
+
+        new_message.save()
+        return new_message
+
+    def get_all_dogowner_contacts(user_id):
+        day_cares = []
+        for msg in Message.objects.filter(dogowner_id=user_id):
+            day_cares.append(msg.daycare_id)
+        return list(set(day_cares))
+
+    def get_all_daycare_contacts(user_id):
+        dog_owners = []
+        for msg in Message.objects.filter(daycare_id=user_id):
+            dog_owners.append(msg.dogowner_id)
+        return list(set(dog_owners))
+
+    def get_chat_between_dogowner_daycare(dogowner_id, daycare_id):
+        return Message.objects.filter(dogowner_id=dogowner_id, daycare_id=daycare_id)
```

Comment:
Looking at this again I find it kind of out of this PR scope, maybe it should be introduced by the PR 
that actually make use of it, for example presenting daycare/dogowner contact.
Not that there is benefit from smaller PR's, they are easier to review eventually..
Let me know what you think, @Yuval-Vino @Omeramsc

Reply:
thanks for the comment Or!  I think it should be in this PR because those are the only methods we need to message class
without them, there is nothing except fields there.
Also, I think we are close to starting the message page so those will become a handful very soon.
I've upgraded the test also
let me know what you think

Reply:
I agree with Or, but I think that since most of it is already done, we can slide it in this time around. 
for later PRs, please create those methods only when you need to use them.